prime-angular-frontend/src/app/modules/site-registration/shared/services/site-form-state.service.ts

```diff
@@ -261,12 +270,49 @@ export class SiteFormStateService extends AbstractFormState<Site> {
 
   private buildHoursOperationForm(): FormGroup {
     return this.fb.group({
-      businessDays: this.fb.array(
-        [],
-        [Validators.required])
+      businessDays: this.fb.array([
```

Comment:
We'll probably want to have validation checking that `atLeaseOne` set of hours has been entered by the user.  It doesn't need to be on the form, but should be checked at least `onSubmit` to prevent submission.  Currently we can submit without choosing hours.